### PR TITLE
Project detail ECC Page and minor bug fixes

### PR DESCRIPTION
**Pull request by deo002**

### CODE REVIEW

**Review by heliocastro — CHANGES_REQUESTED** [review]
Some small requests inline.
Maybe rename Ecc to ECC in all places ?

File: src/app/[locale]/projects/components/Projects.tsx [hunk 1]
```diff
@@ -297,7 +297,13 @@ function Project() {
                         </div>
                         <div className='col-auto buttonheader-title'>{t('PROJECTS')}</div>
                     </div>
-                    <Table columns={columns} server={server} selector={true} sort={false} />
+                    {status === 'authenticated' ? (
```
[the remark this answers is not in the document]
Comment (hoangnt2) [reply src/app/[locale]/projects/components/Projects.tsx:300]: @heliocastro, useSession take a few seconds to retrieve authentication status so we need to check this where useSession is used

[the remark this answers is not in the document]
Comment (heliocastro) [reply src/app/[locale]/projects/components/Projects.tsx:300]: Maybe we're looking wrong and we should protect the routes in different way.
I will approve this one and we try to do something similar like this:
https://www.freecodecamp.org/news/how-to-setup-authentication-and-protected-route-in-next-js-13-with-next-auth-js/

**Review by heliocastro — APPROVED** [review]
Approved with a TODO:
Revise route protection in system

File: src/app/[locale]/projects/components/Projects.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (hoangnt2) [reply src/app/[locale]/projects/components/Projects.tsx:300]: @heliocastro, actually we have route protection in our application (middleware) and it works fine.
What I mean is, when we use the useSession() hook, the state is always **unauthenticated** at the initial time. After a few seconds, the state will be updated (real state) and the component will be re-render.
That's why I get session on server side and send it to client component as a prop.

**Pull request merged**